client/src/components/App/Content/InvestigationForm/InvestigationForm.tsx

```diff
@@ -83,12 +83,18 @@ const InvestigationForm: React.FC = (): JSX.Element => {
             if(isInvestigationValid()) {
                 confirmFinishInvestigation(epidemiologyNumber);
             } else {
+                const displayedForms = [...formsValidations];
+                if (!areThereContacts) displayedForms.pop();
```

Comment:
I don't think pop is the best way to remove the item.
Suppose another tab is added, it will ruin the validation